Emby.Server.Implementations/LiveTv/Listings/XmlTvListingsProvider.cs

```diff
@@ -146,6 +147,16 @@ private string ExtractGz(string file)
             }
         }
 
+        private string GetTempFolderPath(Stream stream)
+        {
+#pragma warning disable CA5351
+            using var md5 = MD5.Create();
+#pragma warning restore CA5351
+            var checksum = Convert.ToHexString(md5.ComputeHash(stream));
```

Comment:
I think this would work too with less overhead
```suggestion
            var checksum = Convert.ToHexString(MD5.HashData(stream));
#pragma warning restore CA5351
```

Reply:
Nvm, this is a .Net 7 API